src/WebJobs.Script/IO/AutoRecoveringFileSystemWatcher.cs

```diff
@@ -131,23 +131,16 @@ private void Trace(string message, TraceLevel level)
 
         private async Task Recover(int attempt = 1)
         {
-            // Exponential backoff on retries
-            long wait = Convert.ToInt64((Math.Pow(2, attempt) - 1) / 2);
-
-            // maximum wait of 5 minutes
-            wait = Math.Min(wait, 300);
-
-            if (wait > 0)
-            {
-                Trace($"Next recovery attempt in {wait} seconds...", TraceLevel.Warning);
```

Comment:
Yeah, I couldn't think of a good way to keep it. In the end, we really only need logs that let us know we're retrying though, which we still have.